06.Artifact-creation/03.Containerized-application-updates/docs.md

```diff
@@ -36,6 +36,10 @@ dependencies are installed on the device:
 !!! docker compose version
 !!! ```
 
+Before generating the artifact, you need to ensure that the following
+dependencies are installed on your **workstation**:
+ * [Mender-artifact tool](../../10.Downloads#mender-artifact)
+ * [app-gen script](https://github.com/mendersoftware/app-update-module/tree/master/gen)
```

Comment:
This is just repeating what's in the following section. Having this here just changes the attention from the device to the workstation and then later back to the workstation. The idea was to make the user journey more like:
1. Prepare the device (client)
2. Prepare your workstation
3. Generate the artifact
4. Deploy it